### PR TITLE
Delete chat system

### PR DESCRIPTION
**Pull request by nabucopena**
This PR deletes the current chat system

### CODE REVIEW

File: migrations/6_create_view_mygames.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+create view my_games as (
```
Comment (carlosipe) [inline migrations/6_create_view_mygames.sql:1]: I think this file doesn't belong to this commit...

Comment (nabucopena) [reply migrations/6_create_view_mygames.sql]: yes, that commit was there by mistake, i have deleted it
